release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -976,6 +992,25 @@ If you are receiving pod security violations, see the following resources:
 
 * If necessary, you can set a custom admission profile on the namespace or pod by setting the `pod-security.kubernetes.io/enforce` label.
 
+[discrete]
+[id="ocp-4-12-olm-catalogs-and-psa-restricted-enforcement"]
+==== Catalog sources and restricted pod security admission enforcement
+
+Catalog sources built using the SQLite-based catalog format and a version of the `opm` CLI tool released before {product-title} 4.11 cannot run under restricted pod security enforcement.
+
+In {product-title} {product-version}, namespaces do not have restricted pod security enforcement by default and the default catalog source security mode is set to `legacy`.
+
+If you do not want to run your SQLite-based catalog source pods as restricted, you do not need to update your catalog source in {product-title} {product-version}. However, it is recommended that you take action now to ensure your catalog sources run under restricted pod security enforcement. If you do not take action to ensure your catalog sources run under restricted pod security enforcement, your catalog sources might not run in future {product-title} releases.
+
+As a catalog author, you can enable compatibility with restricted pod security enforcement by completing either of the following actions:
+
+* Migrate your catalog to the file-based catalog format.
+* Update your catalog image with a version of the `opm` CLI tool released with {product-title} 4.11 or later.
+
+If you do not want to update your SQLite database catalog image or migrate your catalog to the file-based catalog format, you can configure your catalog to run with elevated permissions.
+
+For more information, see xref:../operators/admin/olm-managing-custom-catalogs.adoc#olm-catalog-sources-and-psa_olm-managing-custom-catalogs[Catalog sources and pod security admission].
```

Comment:
The heading this xref takes me to doesn't match the displayed link text here, not sure if it's an issue but just noting it